api/turing/models/router_version.go

```diff
@@ -3,6 +3,7 @@ package models
 import (
 	"database/sql"
 
+	routerConfig "github.com/caraml-dev/turing/engines/router/missionctl/config"
```

Comment:
nit: Separate block for importing Turing-related components. Comment should apply to all imports in the PR.

Reply:
Hmm, I think it's currently not followed in all code. This should be enforced by
```
goimports -w -local github.com/caraml-dev/turing .
```
running the above command will create quite a lot of changes unrelated to the PR. Shall it be done in separate PR instead?

Reply:
https://github.com/caraml-dev/turing/pull/276